marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/customStep/getCustomStep.sjs

```diff
@@ -22,9 +22,9 @@ let dataHubOptions = ["sourceQuery", "sourceQueryIsScript", "constrainSourceQuer
 let additionalSettings = {};
 let step = Artifacts.getArtifact("custom", stepName)
 let options = Artifacts.convertStepReferenceToInlineStep(step.stepId).options;
-for (let [key, value] of Object.entries(options)) {
+for (let key of Object.keys(options)) {
```

Comment:
Also, Object.entries is not available in ML 9.

Reply:
I'm thinking we should make a list of the strings that won't work in ML 9, and we can then have a JUnit test that scans each SJS file and throws an error if any are found. So far, we have at least Object.entries and "{...", right? I can write up a JIRA ticket for it, and we can always add to the list.